docs/components/sensor/bme280.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Configure a bme280 sensor"
+linkTitle: "bme280"
+weight: 70
+type: "docs"
+description: "Configure a bme280 sensor."
+tags: ["sensor", "components"]
+icon: "img/components/sensor.png"
+# SME: #team-bucket
+---
+
+Configure a `bme280` sensor to integrate a [BME280 environmental sensor](https://www.adafruit.com/product/2652), used for sensing temperature, humidity, and barometric pressure, into your robot.
+
+Configure a `bme280` sensor as follows:
+
+{{< tabs >}}
+{{% tab name="Config Builder" %}}
```

Comment:
in the config buider tab you should instruct people to paste the attributes document into the field for it. You can have the full json for the sensor component in a separate tab

Reply:
Oh I think I must've looked at the wrong thing I thought the entire json confit from the other tab was here.

Reply:
However adding a json snippet of what should be in the attributes field seems like what we should do

Reply:
@npentrel Have added across these models